bank_integration/bank_integration/api/hdfc_bank_api.py

```
@@ -232,191 +296,330 @@ def login_success(self):
+                            '//button[contains(@class, "bb-button-bar__button") and contains(@class, "btn-primary") and normalize-space(text())="Get OTP"]',
+                        )
+                    ),
                     EC.visibility_of_element_located((By.NAME, "fldAnswer")),
                     EC.visibility_of_element_located(
-                        (By.XPATH, "//span[@class='successIcon']")
+                        (By.CSS_SELECTOR, "span.success-tick")
                     ),
                 ),
                 throw=False,
             )
         except:
```

Ruff (0.15.4)

[error] Do not use bare `except`

(E722)